code/game/objects/items/theft_tools.dm

```diff
@@ -11,6 +11,7 @@
 	icon_state = "plutonium_core"
 	item_state = "plutoniumcore"
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	layer = 5
```

Comment:
Use the defines for layers rather than a magic number

Reply:
sure, I'll change the SM layer too.